src/main/webapp/search.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Play Even Harder</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <link rel="stylesheet" href="/css/nav-bar.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link href="https://fonts.googleapis.com/css?family=Mali&display=swap" rel="stylesheet">
+  <script src="/js/navigation-loader.js"></script>
+  <script>
+
+    const urlParams = new URLSearchParams(window.location.search);
+    const pattern = urlParams.get('q')
+
+    /** Fetches users and adds them to the page. */
+    function fetchUserList() {
+      const url = '/search?q=' + pattern;
+      fetch(url).then((response) => {
+        return response.json();
+      }).then((users) => {
+        const list = document.getElementById('list');
+        list.innerHTML = '';
+
+        users.forEach((user) => {
+          const userListItem = buildUserListItem(user);
+          list.appendChild(userListItem);
+        });
+      });
+    }
+
+    /**
+     * Builds a list element that contains a link to a user page, e.g.
+     * <li><a href="/user-page.html?user=test@example.com">test@example.com</a></li>
+     */
+    function buildUserListItem(user) {
+      const userLink = document.createElement('a');
+      userLink.setAttribute('href', '/user-page.html?user=' + user);
+      userLink.appendChild(document.createTextNode(user));
+      const userListItem = document.createElement('li');
+      userListItem.appendChild(userLink);
+      return userListItem;
+    }
+
+    /** Fetches data and populates the UI of the page. */
+    function buildUI() {
+      fetchUserList();
+      addLoginOrLogoutLinkToNavigation();
+    }
+  </script>
+</head>
+
+<body onload="buildUI()">
```

Comment:
I think removing onload and then create separate js for fetchUserList() and then added window.onLoad=fetchUserList();
at the end of the js file would be better. 

I have removed all onload are body and add window.onload at the end of necessary js files.
https://github.com/team35-codeu-summer2019/team35-codeu/pull/48#pullrequestreview-254689586